ownership/templates/ownership/login.html

```diff
@@ -0,0 +1,35 @@
+{% extends "ownership/base.html" %}
+
+{% block content %}
+<h4>Login</h4>
+{% if form.errors %}
+<p>Your username and password didn't match. Please try again.</p>
+{% endif %}
+
+{% if next %}
+    {% if user.is_authenticated %}
+    <h4>Your account doesn't have access to this page. To proceed,
+    please login with an account that has access.</h4>
+    {% else %}
+    <h4>Please login to see this page.</h4>
+    {% endif %}
```

Comment:
Oh yeah, I did have these as `<p>` tags initially, but then I added the `<h4>` so now everything's huge!